cmd/bb_runner/BUILD.bazel

```diff
@@ -46,6 +46,24 @@ container_layer(
     files = ["@com_github_krallin_tini_tini_static_amd64//file"],
 )
 
+container_layer(
+    name = "install_layer",
+    files = ["install"],
+)
+
+container_image(
+    name = "bb_runner_installer",
+    base = "@busybox//image",
+    cmd = ["/bb"],
+    entrypoint = ["/install"],
+    layers = [
+        ":install_layer",
+        ":bb_runner_layer",
+        ":tini_layer",
```

Comment:
Oh man. Life would eventually be so much nicer if we didn't need tini. Right now it's only there to prevent zombie process leakage. Reaping those zombies could trivially be done by bb_runner, if it weren't for the fact that that wouldn't be able to interact with cmd.Run() properly. :-/